### PR TITLE
[DPR2-579] Add DPR to navigation

### PR DESCRIPTION
**Pull request by stalwartsolutions**

### CODE REVIEW

File: package-lock.json
Comment (whitfield-mj) [inline package-lock.json:1]: Don't think there should be package-lock changes

Comment (stalwartsolutions) [reply package-lock.json:1]: I think that's just from where I ran `npm install`. I'll roll them back.

Comment (stalwartsolutions) [reply package-lock.json:1]: Done.

**Review by whitfield-mj — APPROVED** [review]
I don't think we really want to get into the habit of introducing legacy approaches, but seeing as you have done it and it will get you started I have approved. When you want to start changing the prisons this is available at we will need to go down the /info route which we will be happy to help with

**Review by davidmiddletonmoj — APPROVED** [review]

**Pull request merged**